util/types/datum.go

```diff
@@ -38,11 +38,11 @@ const (
 	KindFloat64       byte = 4
 	KindString        byte = 5
 	KindBytes         byte = 6
-	KindMysqlBit      byte = 7
+	KindBinString     byte = 7 // Used for BIT / HEX literals.
```

Comment:
`KindBinString` is a little confusing, can we change it to a more meaning name ? maybe `KindBinaryLiteral` ?

Reply:
ok